test/integration/sign_in_test.rb

```diff
@@ -69,6 +69,7 @@ class SignInTest < SystemTest
     fill_in "Password", with: PasswordHelpers::SECURE_TEST_PASSWORD
     click_button "Sign in"
 
+    StatsD.expects(:distribution)
```

Comment:
I decided to just set up an expectation that StatsD would be called. I didn't assert the arguments here since I think it's not necessarily to make assertions on the time being captured (duration). We have that covered in the unit test, so I think it's better to not need to do any "time travelling" in the integration test. WDYT?

Reply:
Fair point, I agree that the important part is to check that the duration is recorded and not necessarily the duration value 👍